external/loaders/tests/test__nudged.py

```diff
@@ -231,10 +238,8 @@ def fine_url(tmpdir):
     return fine_url
```

Comment:
can this `fine_url` fixture be deleted now?

Reply:
yes. done.